composer.json

```diff
@@ -9,8 +9,15 @@
         }
     ],
     "require": {
-        "marco-c/wp_serve_file": "dev-master"
+        "marco-c/wp_serve_file": "dev-master",
+        "stevegrunwell/wp-enforcer": "^0.4.1"
     },
+    "post-install-cmd": [
+    	"wp-enforcer"
```

Comment:
By the way, I think this won't work unless you have `wp-enforcer` installed globally. I think the correct command is `./vendor/bin/wp-enforcer`. What do you think?